spec/helpers/application_helper/buttons/view_spec.rb

```diff
@@ -31,11 +31,11 @@
     before { subject.calculate_properties }
 
     context 'when gtl_type is set' do
-      it_behaves_like 'a disabled button'
+      include_examples 'ApplicationHelper::Button::Basic#disabled?', nil
```

Comment:
Yes, you're right. However, it wouldn't match the first condition, thus the execution would continue in the "enabled" branch, whereas the button is "disabled" but with no error message.
```ruby
if err_message
  it 'is disabled' do
    expect(subject[:enabled]).to be_falsey
    expect(subject[:title]).to eq(err_message)
  end
else
  it 'is enabled' do
    expect(subject[:enabled]).to be_truthy
    expect(subject[:title]).to be_nil
  end
end
```
What could be changed is the first condition as follows:
```ruby
if error_message.nil? || error_message
  ...
end
```
But, what I think is that having a disabled state without a title explaining the reason behind it, would go against the "standard" that is followed by all the other button classes. Having said that, I think that in case of the View button it is a mistake not having such a title declared in its `#disabled?` method.